scripts/upload-ami

```diff
@@ -61,6 +99,40 @@ def upload_to_cloud(buildid, region, bucket, suffix=None, public=False):
         print("Failed to build/upload image! Error: {}".format(upload_err))
         raise
 
+    # If the Aliyun image isn't public, grant access to CI/DEV/QE accounts
+    # TODO: add `--grant-user` support to Aliyun in `ore`
+    if not public and cloud == ALIYUN:
+        arch = subprocess.check_output(["cosa", "basearch"], text='UTF-8').strip()
+        image_id = find_image_id(cloud, region, buildid, arch)
+        share_cmd = ['aliyun', '--endpoint', get_aliyun_endpoint(region), 'ecs',
+                      'ModifyImageSharePermission', '--region', region,
+                      '--ImageId', image_id]
+
+        # The aliyun CLI will return an error if you try to share an image with
+        # your own account, so this is a hack to prevent that from happening
+        # in a RHCOS devel pipeline scenario
+        aliyun_accts = [OPENSHIFT_ALIYUN_CI, OPENSHIFT_ALIYUN_DEV, OPENSHIFT_ALIYUN_QE]
```

Comment:
I think it'd be more obvious to have this be a constant at the top; otherwise someone adding a new account e.g. `_STAGE` will also need to remember to hunt down and append to this list in the middle of the code.